widget/topPanelWidget.js

```diff
@@ -48,7 +47,6 @@ var TopPanel = GObject.registerClass(
             this.tilingButton = new MatButton({
                 child: this.tilingIcon,
                 style_class: 'mat-panel-button',
-                reactive: true,
                 can_focus: true,
                 track_hover: true
```

Comment:
I also removed `can_focus` and `track_hover` while testing and found that it didn't change anything. I'm decently sure that these params can also be removed, but curious what others think.